<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>375</width>
    <height>616</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>FFB Joystick Settings</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <item>
     <widget class="QTabWidget" name="tabWidget_3">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Settings</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QCheckBox" name="enableConstantViaSinCheckBox">
          <property name="toolTip">
           <string/>
          </property>
          <property name="text">
           <string>Enable constant via sine</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="updateDurationOnEffectReportCheckBox">
          <property name="text">
           <string>Update duration on effect report</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="invertDirection_1">
          <property name="text">
           <string>Invert direction</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <item row="1" column="1">
           <widget class="QSlider" name="steeringSmoothingSlider_1">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_27">
            <property name="text">
             <string>Steering smoothing</string>
            </property>
           </widget>
          </item>
          <item row="9" column="0">
           <widget class="QLabel" name="label_15">
            <property name="text">
             <string>Triangle</string>
            </property>
           </widget>
          </item>
          <item row="11" column="2">
           <widget class="QSpinBox" name="sawtoothDownSpinner_1">
            <property name="minimum">
             <number>-100</number>
            </property>
            <property name="maximum">
             <number>100</number>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QSlider" name="rampSlider_1">
            <property name="minimum">
             <number>-100</number>
            </property>
            <property name="maximum">
             <number>100</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="label_26">
            <property name="text">
             <string>Constant</string>
            </property>
           </widget>
          </item>
          <item row="11" column="1">
           <widget class="QSlider" name="sawtoothDownSlider_1">
            <property name="minimum">
             <number>-100</number>
            </property>
            <property name="maximum">
             <number>100</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QSlider" name="torqueGainSlider_1">
            <property name="minimum">
             <number>-100</number>
            </property>
            <property name="maximum">
             <number>100</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="4" column="2">
           <widget class="QSpinBox" name="constantViaSineSpinner_1">
            <property name="minimum">
             <number>-100</number>
            </property>
            <property name="maximum">
             <number>100</number>
            </property>
           </widget>
          </item>
          <item row="7" column="1">
           <widget class="QSlider" name="squareSlider_1">
            <property name="minimum">
             <number>-100</number>
            </property>
            <property name="maximum">
             <number>100</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="6" column="2">
           <widget class="QSpinBox" name="rampSpinner_1">
            <property name="minimum">
             <number>-100</number>
            </property>
            <property name="maximum">
             <number>100</number>
            </property>
           </widget>
          </item>
          <item row="7" column="2">
           <widget class="QSpinBox" name="squareSpinner_1">
            <property name="minimum">
             <number>-100</number>
            </property>
            <property name="maximum">
             <number>100</number>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_21">
            <property name="text">
             <string>Torque smoothing</string>
            </property>
           </widget>
          </item>
          <item row="8" column="0">
           <widget class="QLabel" name="label_22">
            <property name="text">
             <string>Sine</string>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="label_17">
            <property name="text">
             <string>Ramp</string>
            </property>
           </widget>
          </item>
          <item row="11" column="0">
           <widget class="QLabel" name="label_20">
            <property name="text">
             <string>Sawtooth down</string>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QSpinBox" name="torqueSmoothingSpinner_1"/>
          </item>
          <item row="5" column="1">
           <widget class="QSlider" name="constantSlider_1">
            <property name="minimum">
             <number>-100</number>
            </property>
            <property name="maximum">
             <number>100</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="7" column="0">
           <widget class="QLabel" name="label_18">
            <property name="text">
             <string>Square</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_25">
            <property name="text">
             <string>Torque gain</string>
            </property>
           </widget>
          </item>
          <item row="10" column="1">
           <widget class="QSlider" name="sawtoothUpSlider_1">
            <property name="minimum">
             <number>-100</number>
            </property>
            <property name="maximum">
             <number>100</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSlider" name="steeringRangeSlider_1">
            <property name="maximum">
             <number>1000</number>
            </property>
            <property name="singleStep">
             <number>10</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="8" column="1">
           <widget class="QSlider" name="sineSlider_1">
            <property name="minimum">
             <number>-100</number>
            </property>
            <property name="maximum">
             <number>100</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_24">
            <property name="text">
             <string>Constant via sine</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QSpinBox" name="torqueGainSpinner_1">
            <property name="minimum">
             <number>-100</number>
            </property>
            <property name="maximum">
             <number>100</number>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_23">
            <property name="text">
             <string>Steering range</string>
            </property>
           </widget>
          </item>
          <item row="12" column="0">
           <widget class="QLabel" name="label_28">
            <property name="text">
             <string>Spring</string>
            </property>
           </widget>
          </item>
          <item row="5" column="2">
           <widget class="QSpinBox" name="constantSpinner_1">
            <property name="minimum">
             <number>-100</number>
            </property>
            <property name="maximum">
             <number>100</number>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QSlider" name="constantViaSineSlider_1">
            <property name="minimum">
             <number>-100</number>
            </property>
            <property name="maximum">
             <number>100</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QSlider" name="torqueSmoothingSlider_1">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="9" column="1">
           <widget class="QSlider" name="triangleSlider_1">
            <property name="minimum">
             <number>-100</number>
            </property>
            <property name="maximum">
             <number>100</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="12" column="2">
           <widget class="QSpinBox" name="springSpinner_1">
            <property name="minimum">
             <number>-100</number>
            </property>
            <property name="maximum">
             <number>100</number>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QSpinBox" name="steeringSmoothingSpinner_1"/>
          </item>
          <item row="10" column="2">
           <widget class="QSpinBox" name="sawtoothUpSpinner_1">
            <property name="minimum">
             <number>-100</number>
            </property>
            <property name="maximum">
             <number>100</number>
            </property>
           </widget>
          </item>
          <item row="12" column="1">
           <widget class="QSlider" name="springSlider_1">
            <property name="minimum">
             <number>-100</number>
            </property>
            <property name="maximum">
             <number>100</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QSpinBox" name="steeringRangeSpinner_1">
            <property name="maximum">
             <number>1000</number>
            </property>
            <property name="singleStep">
             <number>10</number>
            </property>
           </widget>
          </item>
          <item row="9" column="2">
           <widget class="QSpinBox" name="triangleSpinner_1">
            <property name="minimum">
             <number>-100</number>
            </property>
            <property name="maximum">
             <number>100</number>
            </property>
           </widget>
          </item>
          <item row="10" column="0">
           <widget class="QLabel" name="label_16">
            <property name="text">
             <string>Sawtooth up</string>
            </property>
           </widget>
          </item>
          <item row="8" column="2">
           <widget class="QSpinBox" name="sineSpinner_1">
            <property name="minimum">
             <number>-100</number>
            </property>
            <property name="maximum">
             <number>100</number>
            </property>
           </widget>
          </item>
          <item row="13" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Center</string>
            </property>
           </widget>
          </item>
          <item row="13" column="1">
           <widget class="QPushButton" name="mechanicalCenterSetButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Set to current position</string>
            </property>
           </widget>
          </item>
          <item row="13" column="2">
           <widget class="QSpinBox" name="mechanicalCenterSpinBox">
            <property name="maximum">
             <number>99999</number>
            </property>
            <property name="singleStep">
             <number>10</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout"/>
        </item>
        <item>
         <widget class="QPushButton" name="readButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Read from controller</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="writeButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Wright to controller</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="saveSettingsButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Save</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Advanced</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QCheckBox" name="checkBoenableSystemSettingsCheckBox">
          <property name="text">
           <string>Enable electrical curve calibration</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="calibrateElectricalCenterButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Start electrical poles calibration</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="saveAdvancedSettingsButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Save electrical poles calibration</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>squareSlider_1</sender>
   <signal>valueChanged(int)</signal>
   <receiver>squareSpinner_1</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>294</x>
     <y>327</y>
    </hint>
    <hint type="destinationlabel">
     <x>351</x>
     <y>326</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sawtoothDownSpinner_1</sender>
   <signal>valueChanged(int)</signal>
   <receiver>sawtoothDownSlider_1</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>351</x>
     <y>438</y>
    </hint>
    <hint type="destinationlabel">
     <x>294</x>
     <y>439</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>torqueGainSpinner_1</sender>
   <signal>valueChanged(int)</signal>
   <receiver>torqueGainSlider_1</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>351</x>
     <y>186</y>
    </hint>
    <hint type="destinationlabel">
     <x>294</x>
     <y>187</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>springSlider_1</sender>
   <signal>valueChanged(int)</signal>
   <receiver>springSpinner_1</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>294</x>
     <y>467</y>
    </hint>
    <hint type="destinationlabel">
     <x>351</x>
     <y>466</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>triangleSlider_1</sender>
   <signal>valueChanged(int)</signal>
   <receiver>triangleSpinner_1</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>294</x>
     <y>383</y>
    </hint>
    <hint type="destinationlabel">
     <x>351</x>
     <y>382</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sawtoothUpSlider_1</sender>
   <signal>valueChanged(int)</signal>
   <receiver>sawtoothUpSpinner_1</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>294</x>
     <y>411</y>
    </hint>
    <hint type="destinationlabel">
     <x>351</x>
     <y>410</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>torqueSmoothingSpinner_1</sender>
   <signal>valueChanged(int)</signal>
   <receiver>torqueSmoothingSlider_1</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>351</x>
     <y>214</y>
    </hint>
    <hint type="destinationlabel">
     <x>294</x>
     <y>215</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>steeringRangeSlider_1</sender>
   <signal>valueChanged(int)</signal>
   <receiver>steeringRangeSpinner_1</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>294</x>
     <y>131</y>
    </hint>
    <hint type="destinationlabel">
     <x>351</x>
     <y>130</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rampSpinner_1</sender>
   <signal>valueChanged(int)</signal>
   <receiver>rampSlider_1</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>351</x>
     <y>298</y>
    </hint>
    <hint type="destinationlabel">
     <x>294</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>squareSpinner_1</sender>
   <signal>valueChanged(int)</signal>
   <receiver>squareSlider_1</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>351</x>
     <y>326</y>
    </hint>
    <hint type="destinationlabel">
     <x>294</x>
     <y>327</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sineSlider_1</sender>
   <signal>valueChanged(int)</signal>
   <receiver>sineSpinner_1</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>294</x>
     <y>355</y>
    </hint>
    <hint type="destinationlabel">
     <x>351</x>
     <y>354</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>constantViaSineSpinner_1</sender>
   <signal>valueChanged(int)</signal>
   <receiver>constantViaSineSlider_1</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>351</x>
     <y>242</y>
    </hint>
    <hint type="destinationlabel">
     <x>294</x>
     <y>243</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>steeringRangeSpinner_1</sender>
   <signal>valueChanged(int)</signal>
   <receiver>steeringRangeSlider_1</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>351</x>
     <y>130</y>
    </hint>
    <hint type="destinationlabel">
     <x>294</x>
     <y>131</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoenableSystemSettingsCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>calibrateElectricalCenterButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>60</x>
     <y>56</y>
    </hint>
    <hint type="destinationlabel">
     <x>352</x>
     <y>85</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>steeringSmoothingSpinner_1</sender>
   <signal>valueChanged(int)</signal>
   <receiver>steeringSmoothingSlider_1</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>351</x>
     <y>158</y>
    </hint>
    <hint type="destinationlabel">
     <x>294</x>
     <y>159</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>torqueSmoothingSlider_1</sender>
   <signal>valueChanged(int)</signal>
   <receiver>torqueSmoothingSpinner_1</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>294</x>
     <y>215</y>
    </hint>
    <hint type="destinationlabel">
     <x>351</x>
     <y>214</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sawtoothUpSpinner_1</sender>
   <signal>valueChanged(int)</signal>
   <receiver>sawtoothUpSlider_1</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>351</x>
     <y>410</y>
    </hint>
    <hint type="destinationlabel">
     <x>294</x>
     <y>411</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>constantSpinner_1</sender>
   <signal>valueChanged(int)</signal>
   <receiver>constantSlider_1</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>351</x>
     <y>270</y>
    </hint>
    <hint type="destinationlabel">
     <x>294</x>
     <y>271</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>constantViaSineSlider_1</sender>
   <signal>valueChanged(int)</signal>
   <receiver>constantViaSineSpinner_1</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>294</x>
     <y>243</y>
    </hint>
    <hint type="destinationlabel">
     <x>351</x>
     <y>242</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>triangleSpinner_1</sender>
   <signal>valueChanged(int)</signal>
   <receiver>triangleSlider_1</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>351</x>
     <y>382</y>
    </hint>
    <hint type="destinationlabel">
     <x>294</x>
     <y>383</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>torqueGainSlider_1</sender>
   <signal>valueChanged(int)</signal>
   <receiver>torqueGainSpinner_1</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>294</x>
     <y>187</y>
    </hint>
    <hint type="destinationlabel">
     <x>351</x>
     <y>186</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>springSpinner_1</sender>
   <signal>valueChanged(int)</signal>
   <receiver>springSlider_1</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>351</x>
     <y>466</y>
    </hint>
    <hint type="destinationlabel">
     <x>294</x>
     <y>467</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>constantSlider_1</sender>
   <signal>valueChanged(int)</signal>
   <receiver>constantSpinner_1</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>294</x>
     <y>271</y>
    </hint>
    <hint type="destinationlabel">
     <x>351</x>
     <y>270</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rampSlider_1</sender>
   <signal>valueChanged(int)</signal>
   <receiver>rampSpinner_1</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>294</x>
     <y>299</y>
    </hint>
    <hint type="destinationlabel">
     <x>351</x>
     <y>298</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sineSpinner_1</sender>
   <signal>valueChanged(int)</signal>
   <receiver>sineSlider_1</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>351</x>
     <y>354</y>
    </hint>
    <hint type="destinationlabel">
     <x>294</x>
     <y>355</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sawtoothDownSlider_1</sender>
   <signal>valueChanged(int)</signal>
   <receiver>sawtoothDownSpinner_1</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>294</x>
     <y>439</y>
    </hint>
    <hint type="destinationlabel">
     <x>351</x>
     <y>438</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>steeringSmoothingSlider_1</sender>
   <signal>valueChanged(int)</signal>
   <receiver>steeringSmoothingSpinner_1</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>294</x>
     <y>159</y>
    </hint>
    <hint type="destinationlabel">
     <x>351</x>
     <y>158</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoenableSystemSettingsCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>saveAdvancedSettingsButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>161</x>
     <y>52</y>
    </hint>
    <hint type="destinationlabel">
     <x>161</x>
     <y>101</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
